### ISSUE TITLE
Retweets do not show up in the timeline

### ISSUE THREAD

**Issue by wjdennen** [jfaustin/Tweet-gater#5]
When a user retweets another tweet, it does not show up in the timeline.

**Comment by jfaustin** [comment]
There is an option in the API request that is not present in the ZF library to include RT's.  Since it's not in the ZF library, it's not just something I can immediately fix.  I have, however, submitted a patch to ZF which hopefully will be included in the 11.3 or .4 release.  As soon as that is fixed, I will push an update to take advantage of it.

**Comment by wjdennen** [comment]
thanks.

**Comment by jeanadev** [comment]
Did ZF accept your patch? Just wondering if there's any change in RTs showing.  Thanks.

**Comment by jfaustin** [comment]
Yes they did.  I'll have to check to make sure the code has been updated, but it is in ZF now.
